pages/Plugins/Development/Event-list.md

```diff
@@ -27,8 +27,10 @@ These are all the events that can be listened to using Event Hooks.
 | closeWindow | fired when a window is unmapped | `PHLWINDOW` | ✕ |
 | windowUpdateRules | fired when a window's rules are updated | `PHLWINDOW` | ✕ |
 | urgent | fired when a window requests urgent | `PHLWINDOW` | ✕ |
-| monitorAdded | fired when a monitor is plugged in | `CMonitor*` | ✕ |
-| monitorRemoved | fired when a monitor is unplugged | `CMonitor*` | ✕ |
+| preMonitorAdded | fired when a monitor is plugged in, before Hyprland handles it | `CMonitor*` | ✕ |
```

Comment:
ah all of these CMonitor* should be updated to `PHLMONITOR`. Pls?

Reply:
done